jbpm-console-ng-executor-service/jbpm-console-ng-executor-service-backend/src/main/java/org/jbpm/console/ng/es/backend/server/RemoteExecutorServiceImpl.java

```diff
@@ -28,32 +26,32 @@
 import org.jbpm.console.ng.bd.integration.AbstractKieServerService;
 import org.jbpm.console.ng.es.model.ErrorSummary;
 import org.jbpm.console.ng.es.model.RequestDetails;
-import org.jbpm.console.ng.es.model.RequestKey;
 import org.jbpm.console.ng.es.model.RequestParameterSummary;
 import org.jbpm.console.ng.es.model.RequestSummary;
 import org.jbpm.console.ng.es.service.ExecutorService;
-import org.jbpm.console.ng.ga.model.QueryFilter;
-import org.jbpm.console.ng.ga.service.GenericServiceEntryPoint;
-import org.kie.api.executor.RequestInfo;
-import org.kie.api.executor.STATUS;
 import org.kie.server.api.model.instance.JobRequestInstance;
 import org.kie.server.api.model.instance.RequestInfoInstance;
 import org.kie.server.client.JobServicesClient;
-import org.uberfire.paging.PageResponse;
+
+import static java.util.Collections.emptyList;
+import static java.util.Optional.ofNullable;
+import static java.util.stream.Collectors.toList;
 
 @Service
 @ApplicationScoped
-public class RemoteExecutorServiceImpl extends AbstractKieServerService implements ExecutorService,GenericServiceEntryPoint<RequestKey, RequestSummary> {
+public class RemoteExecutorServiceImpl extends AbstractKieServerService implements ExecutorService {
 
     @Override
     public RequestDetails getRequestDetails(String serverTemplateId, Long requestId) {
         JobServicesClient jobClient = getClient(serverTemplateId, JobServicesClient.class);
 
         RequestInfoInstance request = jobClient.getRequestById(requestId, true, true);
 
-        RequestSummary summary = RequestSummaryHelper.adaptRequest(request);
-        List<ErrorSummary> errors = RequestSummaryHelper.adaptErrorInstanceList(request.getErrors().getItems());
-        List<RequestParameterSummary> params = RequestSummaryHelper.adaptInternalMap(request);
+        RequestSummary summary = ofNullable(request).map(new RequestSummaryMapper()).get();
```

Comment:
I haven't tried, but this doesn't look right.
If `request` is null, `ofNullable(request)` will be empty Optional. Mapping anything over empty Optional results in empty and then empty.get() will result in NoSuchElementException.